Felienne/hedy
# [BUG FIX] Removes achievements pop-up from tutorial

**Description**
In this PR we fix a small issue where achievements would pop-up for new users when following the tutorial. This can be confusing and we prevent this in this PR by simply removing the `onclick()` function on the run button and replace it by the hard-coded output of the example program. As the user can't alter the tutorial program, this output will always be correct.

**Fixes**
This PR fixes #2721.

**How to test**
Make sure you are logged in with an account that doesn't have the **Getting Started I** achievement and navigate to `/tutorial`. Follow the tutorial and verify that no achievement pops up when running the example program. The run counter of the user will also not be increased.
